entities/WealthService.js

```diff
@@ -0,0 +1,29 @@
+module.exports = class WealthService {
+  static increment(player, amount) {
+    player.cash += amount;
+  }
+
+  static decrement(player, amount) {
+    player.cash -= amount;
+  }
+
+  static buyAsset(player, assetPrice) {
```

Comment:
I think we should add a JSDoc comment here that elicits that the developers (i.e., future us) are sure to do a NetWorth Check before calling this function